lib/nettle/pk.c

```diff
@@ -1735,10 +1758,7 @@ _wrap_nettle_pk_verify(gnutls_pk_algorithm_t algo,
 			if (hash_len > vdata->size)
 				hash_len = vdata->size;
 
-			/* SHA-1 is allowed for SigVer in FIPS 140-3 in legacy
-			 * mode */
 			switch (DIG_TO_MAC(sign_params->dsa_dig)) {
-			case GNUTLS_MAC_SHA1:
```

Comment:
are we removing SHA-1?

Reply:
This is SHA1 for signing, not SHA1 in general.